.github/workflows/end-to-end-test.yaml

```diff
@@ -0,0 +1,55 @@
+name: end-to-end-test
+on: push
+jobs:
+  end-to-end:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: checkout the repo
+      uses: actions/checkout@v3
+    - name: build the worker-operator
+      uses: docker/build-push-action@v2
+      with:
+        tags: worker-operator:latest
+        push: false
+    - name: determine node-ip
+      run:  sh .github/workflows/scripts/kind-nodeip.sh
```

Comment:
`determine node-ip` step should be after creating the kind clusters. It is not possible to get node ip before right?